includes/class-linkbacks-handler.php

```diff
@@ -201,29 +201,29 @@ public static function update_meta( $comment_id, $commentdata ) {
 	public static function get_comment_type_excerpts() {
 		$strings = array(
 			// translators: Name verb on domain
-			'mention'       => __( '%1$s mentioned %2$s on <a href="%3$s">%4$s</a>.', 'semantic-linkbacks' ),
+			'mention'         => __( '%1$s mentioned %2$s on <a href="%3$s">%4$s</a>.', 'semantic-linkbacks' ),
```

Comment:
please avoid these changes in the future... it is very noisy

Reply:
I think it was the phpcbf run.